model/gamut_compress.ctl

```diff
@@ -2,17 +2,25 @@
 // <ACESuserName>ACES 1.3 Look - Gamut Compress</ACESuserName>
 
 //
-// "LMT" for compressing out of gamut scene-referred values into AP1.
-// 
-// Input and output are ACES2065-1.
+// Gamut compressor to bring out of gamut scene-referred values into AP1
 //
 
+//
+// Usage:
+//  The intended use of this transform is immediately after the IDT, so that all grading
+//  or compositing operations are downstream of the compression, and therefore work only
+//  with positive AP1 values. It is however not recommended to bake the compression into
+//  VFX pulls, as it may be beneficial for compositors to have access to the unmodified
+//  image data.
 //
 // Direction:
-//    By default this transform operates in the forward direction (compressing the gamut).
```

Comment:
I tend to prefer, "i.e. compressing the gamut" to "(compressing the gamut)", it is usually deemed more appropriate in scientific and research context.

Reply:
Happy to make that change.